### PR TITLE
Add e2e tests for update account password

### PR DESCRIPTION
**Pull request by bibekthapa619**
# Pull Request Template

## Description

This pr adds the e2e tests for update account password feature.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: frontend/tests/acceptance/features/webUIAccount/updatePassword.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    And the user has logged in with email "user1@gmail.com" and password "password"
+    And the user has browsed to the account page
+
+  @blankPasswordUpdate
+  Scenario Outline: update password with blank credentials
+    When the user tries to update password with the following credentials using the webUI
+      | currentPassword   | newPassword   | confirmPassword   |
+      | <currentPassword> | <newPassword> | <confirmPassword> |
+    Then the user should stay on the account page
+    And the user must be able to login with email "user1@gmail.com" and password "password"
```
Comment (saw-jan) [inline frontend/tests/acceptance/features/webUIAccount/updatePassword.feature:17]: checking for empty fields doesn't changes anything. So, this line may not be necessary.

Comment (saw-jan) [inline frontend/tests/acceptance/features/webUIAccount/updatePassword.feature:17]: checking for empty fields doesn't changes anything. So, this line may not be necessary.

Comment (saw-jan) [inline frontend/tests/acceptance/features/webUIAccount/updatePassword.feature:17]:
```suggestion
    
```
checking for empty fields doesn't changes anything. So, this line may not be necessary.

File: frontend/tests/acceptance/features/webUIAccount/updatePassword.feature [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  So that I can log into my account using new password
+
+  Background:
+    Given the user has signed up with name "user1", email "user1@gmail.com" password "password"
+    And the user has logged in with email "user1@gmail.com" and password "password"
+    And the user has browsed to the account page
+
+  @blankPasswordUpdate
+  Scenario Outline: update password with blank credentials
+    When the user tries to update password with the following credentials using the webUI
```
Comment (saw-jan) [inline frontend/tests/acceptance/features/webUIAccount/updatePassword.feature:13]:
```suggestion
    When the user updates password with the following credentials using the webUI
```

File: frontend/tests/acceptance/features/webUIAccount/updatePassword.feature [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    And the user must be able to login with email "user1@gmail.com" and password "password"
+    Examples:
+      | currentPassword | newPassword | confirmPassword |
+      |                 | newPassword | newPassword     |
+      | password        |             | newPassword     |
+      | password        | newpassword |                 |
+
+  @invalidCurrentPasswordUpdate
+  Scenario: update password with invalid current credentials
+    When the user tries to update password with the following credentials using the webUI
```
Comment (saw-jan) [inline frontend/tests/acceptance/features/webUIAccount/updatePassword.feature:26]:
```suggestion
    When the user updates password with the following credentials using the webUI
```

File: frontend/tests/acceptance/features/webUIAccount/updatePassword.feature [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      | currentPassword | newPassword | confirmPassword |
+      | password123     | newPassword | newPassword     |
+    Then the user should be displayed a popup with message "Password mismatch"
+    And the user must be able to login with email "user1@gmail.com" and password "password"
+
+  @unmatchPasswordUpdate
+  Scenario: update password with unmatching new password
+    When the user tries to update password with the following credentials using the webUI
```
Comment (saw-jan) [inline frontend/tests/acceptance/features/webUIAccount/updatePassword.feature:34]:
```suggestion
    When the user updates password with the following credentials using the webUI
```

File: frontend/tests/acceptance/features/webUIAccount/updatePassword.feature [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+      | currentPassword | newPassword | confirmPassword |
+      | password        | newPassword | Password        |
+    Then a password matching error message "Password and confirm password mismatch" should be displayed
```
Comment (saw-jan) [inline frontend/tests/acceptance/features/webUIAccount/updatePassword.feature:37]:
```suggestion
    Then a password mismatch error message "Password and confirm password mismatch" should be displayed
```

**Comment by koushikmohan1996** [comment]
@saw-jan Can you review this?

**Comment by saw-jan** [comment]
@koushikmohan1996  he has made the changes. and the PR is good to be merged.

**Comment by koushikmohan1996** [comment]
Thanks :)

**Pull request merged**